src/features/birthdayNotification/pages/components/BirthdayNotification.tsx

```diff
@@ -0,0 +1,164 @@
+import { useUserProfileInfo } from 'features/home/api'
+import { navigateToHome } from 'features/navigation/helpers'
+import { UseNavigationType } from 'features/navigation/RootNavigator'
+import { analytics } from 'libs/analytics'
+import { storage } from 'libs/storage'
+import TutorialPassLogo from 'ui/animations/eighteen_birthday.json'
+import { ProgressBar } from 'ui/components/bars/ProgressBar'
+import { ButtonPrimary } from 'ui/components/buttons/ButtonPrimary'
+import { useSnackBarContext } from 'ui/components/snackBar/SnackBarContext'
+import { Spacer } from 'ui/components/spacer/Spacer'
+import CategoryIcon from 'ui/svg/icons/categories/bicolor'
+import { ColorsEnum, getSpacing, Typo } from 'ui/theme'
+import { useGrid } from 'ui/theme/grid'
+
+export const BirthdayNotification = () => {
+  const { navigate } = useNavigation<UseNavigationType>()
+  const { data: profile } = useUserProfileInfo()
+  const { showInfoSnackBar } = useSnackBarContext()
+  const depositAmount = useDepositAmount()
+  const deposit = depositAmount.replace(' ', '')
+  const age =
+    new Date().getFullYear() - new Date(profile?.dateOfBirth ?? '2005-10-25').getFullYear()
+  const grid = useGrid()
+  const animationRef = React.useRef<LottieView>(null)
+  const animatedButtonRef = React.useRef<Animatable.View & View>(null)
+  const amount = 30 //TODO: get the real amount
+  const total = 55
+  const progress = total / 80
+
+  function onButtonPress() {
+    if (profile) {
+      navigateToHome()
+    } else {
+      navigate('Login')
+      showInfoSnackBar({
+        message: t`Tu n'es pas connecté !`,
+      })
+    }
+  }
+
+  useEffect(() => {
+    storage.saveObject('has_seen_birthday_notification_card', true)
+  }, [])
+
+  useEffect(() => {
+    if (animationRef.current) {
+      animationRef.current.play(0, 62)
+    }
+  }, [animationRef])
+
+  useEffect(() => {
+    const button = animatedButtonRef?.current
+    if (button?.fadeIn) {
+      button.fadeIn(1000)
+    }
+  }, [animatedButtonRef])
+
+  useEffect(() => {
+    analytics.logScreenView('BirthdayNotification')
+  }, [])
+
+  const amountText = `${amount}€`
+  return (
+    <Container>
+      <Spacer.Flex flex={grid({ sm: 0.2, default: 0.5 }, 'height')} />
+      <StyledLottieContainer height={`${grid({ sm: 20, default: 30 }, 'height')}%`}>
+        <StyledLottieView ref={animationRef} source={TutorialPassLogo} loop={false} />
+      </StyledLottieContainer>
+      <Spacer.Flex flex={grid({ sm: 0.1, default: 0.5 }, 'height')} />
+      <StyledTitle>{t`Bonne nouvelle !`}</StyledTitle>
+      <StyledSubTitle>{t({ id: `Tu as {age} ans...`, values: { age } })}</StyledSubTitle>
+      <Spacer.Flex flex={grid({ sm: 0.1, default: 0.5 }, 'height')} />
+      <StyledBody>
+        {t({
+          id: 'birthday notification text',
+          values: { deposit },
+          message: `Pour tes 16 ans, le Gouvernement ajoute 30 euros supplémentaires à ton crédit pour un total de :`,
```

Comment:
```suggestion
          message: `Pour tes {age} ans, le Gouvernement ajoute {amountText} euros supplémentaires à ton crédit pour un total de :`,
```
ou qlq chose comme ça, non ?

Reply:
Bien vue